platform/darwin/test/MGLCodingTests.mm

```diff
@@ -41,6 +44,36 @@ - (void)testPointFeature {
     XCTAssertEqualObjects(pointFeature, unarchivedPointFeature);
 }
 
+- (void)testPointFeatureCluster {
+    MGLPointFeature *pointFeature = [[MGLPointFeatureCluster alloc] init];
+    pointFeature.title = @"title";
+    pointFeature.subtitle = @"subtitle";
+    pointFeature.identifier = @(123);
+    pointFeature.attributes = @{
+        @"cluster" : @(YES),
+        @"cluster_id" : @(456),
+        @"point_count" : @(2),
+        };
```

Comment:
Super-nit: the indentation here is SLIGHTLY OFF. 🙅‍♂️

Reply:
Done